tests/tc_auto_variable/expected_nf.txt

```diff
@@ -0,0 +1,4 @@
+identifiers
+data	v	MyData data	stdin 41	
```

Comment:
I have a feeling this should have rather been `Stuff`, is this related to this change or was this always `MyData` before?

Reply:
It was already ``MyData`` before the change